docs/installation.rst

```diff
@@ -160,13 +160,7 @@ Weekly builds
 ^^^^^^^^^^^^^
 
 Also, there is a weekly release channel with more thorough testing. To access most recent weekly
-builds please use flowing release channel:
-
-* for CUDA 10:
-
-.. code-block:: bash
-
-  pip install --extra-index-url https://developer.download.nvidia.com/compute/redist/weekly nvidia-dali-weekly-cuda100 nvidia-dali-tf-plugin-weekly-cuda100
+builds please use flowing release channel (available only for CUDA 11):
```

Comment:
```suggestion
builds please use the following release channel (available only for CUDA 11):
```
If you already say it's only available for CUDA 11, I am not sure if line 165 still applies

Reply:
Removed